src/pages/Home/HomePage.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import Post from '../../components/Post/Post';
+import charityFinder from "../../assets/charityFinder.png";
+
+const postArray: Post[] = [
```

Comment:
for sample posts, we can move it into a different file to make it organized.

Reply:
@yukfaihsu

Reply:
Moving comment to issues so we can close this PR